modules/rosa-create-objects.adoc

```diff
@@ -116,14 +116,20 @@ $ rosa create cluster --cluster=<cluster_name> | <cluster_id> [arguments]
 |--private
 |Restricts master API endpoint and application routes to direct, private connectivity.
 
+|--private-link
+| Specifies to use AWS PrivateLink to provide private connectivity between VPCs and services. The `--subnet-ids` argument is required when using `--private-link`.
+
 |--region
 |The AWS region (string) where your worker pool will be located. This argument overrides the `AWS_REGION` environment variable.
 
 |--service-cidr
 |Block of IP addresses (ipNet) for services. Example: `172.30.0.0/16`
 
 |--subnet-ids
-|The subnet IDs (string) to use when installing the cluster. Subnet IDs must be in pairs with one private subnet ID and one public subnet ID per availability zone. Subnets are comma-delimited. Example: `--subnet-ids=subnet-1,subnet-2`. Leave the value empty for installer-provisioned subnet IDs.
+|The subnet IDs (string) to use when installing the cluster. Subnet IDs must be in pairs with one private subnet ID and one public subnet ID per availability zone. Subnets are comma-delimited. Example: `--subnet-ids=subnet-1,subnet-2`. Leave the value empty for installer-provisioned subnet IDs. 
+
+
+The `--subnet-ids` argument is required when using `--private-links`.
 
 |--version
 |The version (string) of OpenShift Container Platform that will be used to install the cluster. Example: `4.3.10`
```

Comment:
Should we use `The version (string) of **OpenShift Container Platform** that will be used to install the cluster` or **OpenShift Dedicated Cluster**

Reply:
@jonquilwilliams ^

Reply:
@xueli181114 OpenShift Dedicated Cluster is not correct. The version number is the OCP version that ROSA and OSD are based on. I do not understand what you meant by using **OpenShift Container Platform** instead of OpenShift Container Platform. Are you asking us to put the OCP product name in bold formatting? Please clarify. Thank you.

Reply:
@jonquilwilliams I didn't mean to bold it. Just want to confirm whether OpenShift Dedicated Cluster is better. As you said, OpenShift Dedicated Cluster is not correct. So it's ok to discard the comment